docs/security.rst

```diff
@@ -52,23 +59,52 @@ By default Schema Registry ships with configs that allow you to make REST API ca
   * Importance: high
 
 ``schema.registry.inter.instance.protocol``
-  The protocol used while making calls between the instances of schema registry. The slave to master node calls for writes and deletes will use the specified protocol. The default value would be `http`. When `https` is set, `ssl.keystore.` and `ssl.truststore.` configs are used while making the call.
+  The protocol used while making calls between the instances of Schema Registry. The slave to master node calls for writes and deletes will use the specified protocol. The default value would be `http`. When `https` is set, `ssl.keystore.` and `ssl.truststore.` configs are used while making the call.
 
   * Type: string
   * Default: "http"
   * Importance: low
 
-If you would want to upgrade to HTTPS in an existing cluster, you need to perform the outlined steps:
+On the client, configure the ``schema.registry.listener`` to match the configured Schema Registry listener.
 
-- Add/Modify the ``listeners`` config  to include HTTPS. For example: http://0.0.0.0:8081,https://0.0.0.0:8082
-- Configure the appropriate SSL configurations
 
-    - ``ssl.keystore.*`` configs to setup keystore related configs
-    - ``ssl.truststore.*`` configs to setup truststore related configs (required only when ``ssl.client.auth`` set to true)
-   
+Additional configurations for HTTPS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are using HTTPS, configure the Schema Registry with appropriate SSL configurations for the keystore and optionally truststore. The truststore is required only when ``ssl.client.auth`` set to true.
+
+.. sourcecode:: bash
+
+   ssl.truststore.location=/var/private/ssl/kafka.client.truststore.jks
+   ssl.truststore.password=test1234
+   ssl.keystore.location=/var/private/ssl/kafka.client.keystore.jks
+   ssl.keystore.password=test1234
+   ssl.key.password=test1234
+
+To configure clients to use HTTPS to Schema Registry:
+
+1. On the client, configure the ``schema.registry.listener`` to match the configured listener for HTTPS.
```

Comment:
schema.registry.url instead of schema.registry.listener ??

Reply:
Good catch. Fixed.